src/NuGet.Core/NuGet.Build.Tasks.Pack/Strings.resx

```diff
@@ -130,7 +130,8 @@
     <comment>{0} is the file being packed.</comment>
   </data>
   <data name="InvalidLicenseCombination" xml:space="preserve">
-    <value>Invalid metadata. Cannot specify both a License Expression and a License File.</value>
+    <value>Invalid metadata. Cannot specify both a PackageLicenseExpression and a PackageLicenseFile.</value>
+    <comment>Please don't localize PackageLicenseUrl, PackageLicenseFile and PackageLicenseExpression.</comment>
```

Comment:
nit: `PackageLicenseUrl` is not used in this string, remove it from the comment.